<ui version="4.0" >
 <class>LightPresetsClass</class>
 <widget class="QMainWindow" name="LightPresetsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>800</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>800</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Light Presets</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Master</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>80</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveNewButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>5</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Save New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overWriteButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>22</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Overwrite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>39</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blackButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>63</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fullPowerButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>80</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Full Power</string>
    </property>
   </widget>
   <widget class="QLabel" name="masterStrength" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>0</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="masterTimer" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>80</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>00,0s</string>
    </property>
   </widget>
   <widget class="QDial" name="masterDial" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>20</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="notchesVisible" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="masterSlider" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>18</y>
      <width>21</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>100</number>
    </property>
    <property name="sliderPosition" >
     <number>100</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
